libeos-updater-util/flatpak-util.c

```diff
@@ -1844,6 +1906,8 @@ find_related_refs_for_action (FlatpakInstallation       *installation,
         if (!flatpak_transaction_add_update (transaction, ref_action_ref, NULL, NULL, error))
           return FALSE;
         break;
+      case EUU_FLATPAK_REMOTE_REF_ACTION_PRUNE_REF:
+        break;
```

Comment:
Suggest adding a short comment in here to make it clear this case isn’t accidentally unimplemented.